docs/usage.rst

```diff
@@ -172,9 +172,32 @@ container based on the given image name. This assumes the ``fv3config`` package
 ``fv3gfs`` python wrapper are installed inside the container, along with any
 dependencies.
 
+
 The python interface is very similar to the command-line interface, but is split into
 separate functions based on where the model is being run.
 
+Customizing the model execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please replace this separator with a "----" separator since that's what's used for all other headings.

Reply:
I had intended it to be a sub-section of the one above, but I guess it makes sense as higher-level section too.